package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "react-native-job-queue",
     "title": "React Native Job Queue",
-    "version": "0.4.1",
+    "version": "0.4.2",
```

Comment:
The version gets bumped on master by the release process with `npm version` so you don't have to :-)